### PR TITLE
Animation/get started guide

### PR DESCRIPTION
**Pull request by rodleviton**
screencapture-localhost-8000-guides-animation-2018-11-14-10_41_38

### CODE REVIEW

**Comment by rodleviton** [comment]
[Kaizen Animation - "Getting Started" guide](https://trello.com/c/o8fWT8SC/607-kaizen-animation-getting-started-guide)

File: guide/src/pages/guides/animation/index.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+| `duration` (Keyword)    | Length (speed) of the transition. (Default: `$ca-duration-slow`)                 |
+| `delay` (Duration)      | Delay in seconds or milliseconds before the transition starts. (Default: `null`) |
+
+#### Transitions
+
+To use any of the transition presets, simply define a class and include the mixin. All preset transition mixins are prefixed with `ca-transition-`.
+
+##### Transition Presets
+
+<InlineNotification persistent={true} type="informative">**Note:** Transitions will need to be “triggered” by adding `.ca-enter-active` or `.ca-leave-active` classes.</InlineNotification>
```
Comment (didoesdigital) [inline guide/src/pages/guides/animation/index.md]: @rodleviton any particular reason for `leave` instead of `exit`?

Comment (rodleviton) [reply guide/src/pages/guides/animation/index.md]: Hey @didoesdigital. Less of a conscious decision as this is inherited from the motion-ui presets. We can change this to `exit` if you think it is more intuitive.

**Pull request merged**